src/services/EcoBridge/OmniBridge/OmniBridge.utils.ts

```diff
@@ -580,7 +580,8 @@ export const combineTransactions = (
       txHash,
       assetName: symbol,
       assetAddressL1: token,
-      value: formatUnits(amount, decimals),
+      fromValue: formatUnits(amount, decimals),
+      toValue: '0', // @TODO: add toValue
```

Comment:
Don't know if this is forgotten or to do later 😄

Reply:
Fixed it